src/config/models.rs

```diff
@@ -49,11 +49,15 @@ pub struct ConfigOptsServe {
     #[structopt(long = "proxy-backend")]
     #[serde(default)]
     pub proxy_backend: Option<Url>,
-    /// The URI on which to accept requests which are to be rewritten and proxied to backend
+    /// The URI on which to accept requests which are to be proxied to backend
     /// [default: None]
-    #[structopt(long = "proxy-rewrite")]
+    #[structopt(long = "proxy-path")]
     #[serde(default)]
-    pub proxy_rewrite: Option<String>,
+    pub proxy_path: Option<String>,
+    /// Whether the endpoint for a websocket. [default: false]
```

Comment:
```suggestion
    /// Whether the endpoint for a websocket [default: false]
```
Just because structopt typically strips the trailing `.`, which is what we do everywhere else for the CLI docs as well.